aten/src/ATen/native/Copy.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "ATen/ATen.h"
+
+namespace at {
+namespace native {
+
+// C and C++ have a lovely set of implicit conversion rules, where casting
```

Comment:
Err, shouldn't you delete the old copy of this comment?

Alt, give it a label `Note [Implicit conversion between signed and unsigned]` and just refer to the label here. But copy pasting the comment is a no-no.